dts/bindings/power/state.yaml

```diff
@@ -0,0 +1,26 @@
+# Copyright (c) 2020, Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+description: Properties for power management state
+
+compatible: "pm-state"
+
+properties:
+    pm-state:
+        type: string
+        required: true
+        description: indicates a power state
+        enum:
+            - "active"
+            - "runtime_idle"
```

Comment:
nit-pick: lower case with hyphens.  This is more consistent enum use in Linux bindings.

Reply:
I wasn't going to ask for this, but I did think the same thing, fwiw.

Reply:
done, anything else ?